packages/api/src/period/types.ts

```diff
@@ -124,3 +125,9 @@ export interface ReplaceQuantifierRequestBody {
   currentQuantifierId: string;
   newQuantifierId: string;
 }
+
+export interface SummarizedPeriodData {
```

Comment:
This represents one interface for one custom export only. There will be many export types defined by .. whoever. There will be no export intefaces since everything is based just on the transformer json format.